dbscripts/xml/upgrade.xml

```diff
@@ -69,7 +69,6 @@
 	<schema file="lib/pkp/xml/schema/tombstone.xml" />
 	<schema file="lib/pkp/xml/schema/rolesAndUserGroups.xml" />
 	<schema file="lib/pkp/xml/schema/metrics.xml" />
-	<schema file="lib/pkp/xml/schema/views.xml" />
```

Comment:
I don't see this added somewhere else -- is this intentional? Was this removed because the schema hasn't changed?

Reply:
Ah, I missed that. No, this is just wasted cycles and good to remove.